scripts/assets/file_content_validation_checks.textproto

```diff
@@ -36,6 +36,13 @@ file_content_checks {
   exempted_file_name: "domain/src/main/java/org/oppia/android/domain/oppialogger/logscheduler/MetricLogSchedulingWorker.kt"
   exempted_file_name: "scripts/src/javatests/org/oppia/android/scripts/regex/RegexPatternValidationCheckTest.kt"
 }
+file_content_checks {
+  file_path_regex: ".+?\\.xml"
+  prohibited_content_regex: "<include"
+  failure_message: "Remove <include .../> tag from layout and use the appropriate widget, e.g AppBarLayout widget instead."
+  exempted_file_name: "app/src/main/res/layout/administrator_controls_activity.xml"
+  exempted_file_name: "app/src/main/res/layout-sw600dp/administrator_controls_activity.xml"
```

Comment:
You mentioned in your PR description that fixing these two files results in test failures--what failures specifically? I'm not keen on including initial exemptions like this if we can help it, so it'd be preferable to fix the failures instead of ignoring them.

Reply:
The exempted lines have been removed. The test failures were avoided and so no failures have been ignored with the latest changes. @BenHenning PTAL